tests/conftest.py

```diff
@@ -41,6 +43,21 @@
 )
 
 
+TMP_DIR = "/tmp/pytest_tmp_path/"
+
+# define own tmp_path fixture for older version of pytest (Centos)
+try:
+    from _pytest import tmpdir
+
+    _ = tmpdir.tmp_path
+except (ImportError, AttributeError, KeyError):
+
+    @pytest.fixture()
+    def tmp_path():
+        Path(TMP_DIR).mkdir(exist_ok=True, parents=True)
+        return Path(tempfile.mkdtemp(prefix=TMP_DIR))
```

Comment:
I worry this won't work because tmp_path fixture provides unique paths for tests while this one is static for all - we'll see in the test results.

Reply:
I have no idea if fixture code is executed for every usage in `def ...(tmp_path)` But I hoped so :-)
so seems to work now somehow

Reply:
`scope="function"`, which means to create the fixture for each test function, is the default, but you could specify it to be more clear.